### PR TITLE
Glossary table

### PR DESCRIPTION
**Pull request by elliemdaw**
This table aims to capture terms, meanings, and contexts. I wrote commentary on the importance of getting this right, which might be better as a blog post. This PR includes just the glossary content.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -14,10 +14,24 @@ See the [LFPH Landscape](https://landscape.lfph.io/category=gaen-bluetooth&forma
...
+| **GAEN** | Google/Apple Exposure Notification | You will often see this referring to the GAEN API (Application Programming Interface) or the GAEN Framework, technologies provided by Google and Apple to be used by development teams to build apps with the ability to perform private proximity detection and alerting. |
+| **Implementer** | The team that is responsible for the technical execution of the exposure notification app | Usually, but not always, this is a technology firm hired by the PHA. |
+| **PHA** | Public Health Authority | PHAs are typically associated with the government of a certain locality. |
+| **Proximity** | A measure of distance between items, not associated with location | EN app measure the proximity between smartphones to understand whether a contact event has occurred. This allows notification of potential exposure without having to collect user locations. |
```
Comment (jennydove) [inline docs/index.md]: typo - should be "EN apps measure"

File: docs/index.md [hunk 2]
```diff
@@ -14,10 +14,24 @@ See the [LFPH Landscape](https://landscape.lfph.io/category=gaen-bluetooth&forma
...
+| **LFPH** | Linux Foundation Public Health | As an open source community, Linux Foundation's new Public Health initiative combines efforts of multiple groups working toward privacy-preserving technologies, initially geared toward technology to aid in the fight against COVID-19 |
```
Comment (jennydove) [inline docs/index.md]: needs a period at the end of the sentence

**Comment by jennydove** [comment]
Should add a line for EN Apps since that's a term you use in the other definitions. 

Also this is a little detailed for the index page. Can we move this table to "Getting Started"?

File: docs/index.md [hunk 3]
```diff
@@ -14,10 +14,24 @@ See the [LFPH Landscape](https://landscape.lfph.io/category=gaen-bluetooth&forma
...
+| **Surveillance** | Monitoring or observing activity | Epidemiologists sometimes use this term to refer to their work to understand the spread and containment of a virus. Even though EN apps can aid in containment efforts, privacy-preserving apps do not perform surveillance from a technology perspective as they do not collect location, track, or record user activity. |
```
Comment (jennydove) [inline docs/index.md]: There's a lack of parallelism in this final list - I think it should be "they do not collect, track, or record user location or history." ?

**Review by jennydove — CHANGES_REQUESTED** [review]
Pls move to "getting started" page (probably at the end) and address the few typos/corrections.

**Review by jennydove — APPROVED** [review]

**Comment by jennydove** [comment]
LGTM

**Comment by elliemdaw** [comment]
@jennydove thank you! It doesn't look like I have permissions to merge, would you mind merging? :)

**Pull request merged**